src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -124,653 +126,602 @@ def _configure_db_create_params(
         raise ValueError('Engine {} does not support create mode {}'.format(engine, create_mode))
 
     # Include all Database params as a starting point. We will filter from there.
-    cmd.expand('parameters', Database)
+    arg_ctx.expand('parameters', Database)
```

Comment:
expand leverages .extra so I'm a little confused why it's not working for you.

Reply:
it works now